### PR TITLE
third stage ready

### PR DESCRIPTION
**Pull request by agavriluta**
image

### CODE REVIEW

File: MediumProject/src/main/java/stage3/Menu.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    === Menu ===
+                    1. Find a person
+                    2. Print all people
+                    0. Exit""");
+        return scan.nextLine();
+
+    }
+    private static final Map<String , Consumer<String>> actionsMap;
+    static {
+        Consumer<String> find = (search) -> {
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/stage3/Menu.java]: variables assignment is redundant + use util map methods

        Consumer<String> find = (search) -> {
            String input = scan.nextLine();
            System.out.println("\nEnter a name or email to search all suitable people.");
           SearchingEngine.searchPerson(input);
        };
        Consumer<String> print = (search) -> {
            UsersDataOutput.printAllPeople();
        };
        actionsMap = new HashMap<>();
        actionsMap.put("1" , find);
        actionsMap.put("2" , print);

=>>>

        Map.of(
"1" , 
(search) -> {
            String input = scan.nextLine();
            System.out.println("\nEnter a name or email to search all suitable people.");
           SearchingEngine.searchPerson(input)),
 "2" ,
 (search) -> {
            UsersDataOutput.printAllPeople());

File: MediumProject/src/main/java/Main.java [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import stage1.Searching;
+import stage3.Menu;
+import stage3.UserDetails;
 
 public class Main {
     public static void main(String[] args) {
-        Searching.search();
+       startStage3();
+    }
+    private static void startStage3(){
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/Main.java]: please give appropriate name for the method

**Review by ppanasencoGD — APPROVED** [review]

File: MediumProject/src/main/java/stage3/Menu.java (hunk 1 above)
Comment (agavriluta) [reply MediumProject/src/main/java/stage3/Menu.java]: done

File: MediumProject/src/main/java/Main.java (hunk 2 above)
Comment (agavriluta) [reply MediumProject/src/main/java/Main.java]: done

**Pull request merged**